src/assets/style.css

```diff
@@ -0,0 +1,35 @@
+.bodyApp {
```

Comment:
Hi we have this now:

![Screenshot 2019-12-03 at 8 16 32 AM](https://user-images.githubusercontent.com/10922508/70028760-cf4df000-15a5-11ea-926a-075ed7be359e.png)


This is what our final design should look like:

![Screenshot 2019-12-03 at 8 14 03 AM](https://user-images.githubusercontent.com/10922508/70028785-db39b200-15a5-11ea-8050-546f3893a879.png)

without the missing button names of course.

Reply:
Let's send the color props of grey for all the button except the last in the group and the default props of orange can be used for the rest.